docs/blog/2018-2-7-jam-out-your-blog/index.md

```diff
@@ -73,7 +73,7 @@ Prose was originally created for editing markdown files on GitHub by the people
 ### Here is a summary:
 
 - Simple Content Authoring Environment
-- Designed for CMS-Free Websites
+- designed for CMS-Free Websites
```

Comment:
This seems like it should remain upper-cased since all the other items are upper-cased
```suggestion
- Designed for CMS-Free Websites
```